app/views/govuk_publishing_components/components/_hint.html.erb

```diff
@@ -1,8 +1,9 @@
 <%
   id ||= "hint-#{SecureRandom.hex(4)}"
-  classes ||= ''
+  margin_bottom ||= 3
+
   css_classes = %w( gem-c-hint govuk-hint )
-  css_classes << classes if classes
+  css_classes << ([*0..9].include?(margin_bottom) ? "govuk-!-margin-bottom-#{margin_bottom}" : "govuk-!-margin-bottom-3")
```

Comment:
If/when this moves into other components it could be a helper?

Reply:
Agreed. I've got some more candidates but wanted to see if the initial model would meet with approval first.